.eslintrc

```diff
@@ -85,13 +85,8 @@
     },
     {
       "files": [
-        "app/javascript/packages/address-search/components/in-person-locations.spec.tsx",
-        "app/javascript/packages/components/spinner-dots.jsx",
         "app/javascript/packages/document-capture/components/acuant-capture.tsx",
-        "app/javascript/packages/document-capture/components/acuant-selfie-capture-canvas.jsx",
-        "app/javascript/packages/document-capture/components/document-side-acuant-capture.jsx",
```

Comment:
Excited for these changes!

We can also remove these files in the new TypeScript enforcement exceptions added in #10496:

https://github.com/18F/identity-idp/blob/5c758fd167df84170312ec58ab8daba842d4c730/scripts/enforce-typescript-files.mjs#L8-L76

(Looks like `spinner-dots.jsx` got to sneak past the exception since it's part of a published package)

Reply:
Oh, nice! I'll remove files from that list as I convert them to typescript.